### ISSUE TITLE
0.2.5 and git version doesn't show its windows

### ISSUE THREAD

**Issue by SilverRainZ** [RicinApp/Ricin#149]
- Arch Linux
- GTK 3.22.2+4+gc54f348-1
- Build script: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=ricin

When I execute ricin, it only output the following logs, and doesnt show any window.

```
/ la @ la-arch 18:29 -> ~/pkg/aur/ricin-git git:(master) x 
\ $ ricin
DEBUG: GetText test: This is a sentence in English.
INFO: Ricin version 1.2.0 started !
INFO: Running ToxCore version 0.0.0
DEBUG: Started Tox instance without specified profile

** (ricin:6422): CRITICAL **: ricin_core_profile_construct: assertion 'handle != NULL' failed
DEBUG: ToxID: **************************************
DEBUG: B: Started Tox bootstraping process...
DEBUG: B: dht-nodes.json was found, parsing it.
DEBUG: B: Parsed dht-nodes.json, bootstraping in progress...
DEBUG: B: UDP IPv4 bootstrap 46.38.239.179:33445 by Martin Schröder
DEBUG: B: UDP IPv6 bootstrap v6.tox.zodiaclabs.org:33445 by stal
DEBUG: B: UDP IPv6 bootstrap sorunome.de:33445 by Sorunome
DEBUG: B: UDP IPv6 bootstrap 2a06:8ec0:1:bb::3862:33445 by Kostik
DEBUG: B: UDP IPv4 bootstrap d4rk4.ru:1813 by D4rk4
DEBUG: B: UDP IPv4 bootstrap d4rk4.ru:1813 by D4rk4
...
```

**Comment by SkyzohKey** [comment]
Yeah it seems the PKGBUILD use the bad branch (rewrite-clean instead of master).  
Changine the pkgbuild to compile master should fix the issue x)

**Comment by SkyzohKey** [comment]
Just released 0.2.6 under the correct branch, everything should be fine now!
